### ISSUE TITLE
Road Map For Spring Cloud Releases?

### ISSUE THREAD

**Issue by tkvangorder** [spring-cloud/spring-cloud-starters#45]
Is it possible to bump the spring boot dependency on spring cloud to 1.3.3? 

We are running into this issue in logback:  [http://jira.qos.ch/browse/LOGBACK-730](http://jira.qos.ch/browse/LOGBACK-730)

Spring Boot 1.3.3 depends on logback 1.1.5 (Bug is fixed)
Spring Boot 1.3.2 depends on logback 1.1.3 (where this bug manifests)

For now, we are just overriding the logback dependency in our pom file, but it would be nice to have this in the core once Brixon becomes final. 

Is there an updated roadmap for spring cloud releases?

**Comment by spencergibb** [comment]
https://github.com/spring-cloud/spring-cloud-build/blob/master/pom.xml

I believe this is already done in Brixton. Are you seeing otherwise?

**Comment by tkvangorder** [comment]
We are still on Brixton.M5 and I hadn't look on the master branch. It would be very helpful to have a roadmap page that tracks the releases (and future releases of Spring Cloud and its dependent version of Spring Boot).

**Comment by tkvangorder** [comment]
Is there any sort of time frame or road map for the Spring Cloud releases? We are currently using Brixton.M5 in a production level setting and that feels a bit risky. It would be nice to know when Brixton will become the "official" release.

**Comment by spencergibb** [comment]
We've been waiting on spring-cloud-stream which just released RC1 today. We will finish RC1 today or tomorrow. Then a few weeks of bug fixes. Nothing is set in stone though.
